### DISCUSSION TITLE
how to sum a column from a child table as a summary column in the parent table

### DISCUSSION THREAD

**Discussion by dgagliardi** [Budibase/budibase#5888] [Help] [answered]
Hi, assume I have 2 tables
Tests {id, testName, testDate, totalPoints]
Questions {id, question, description, points}
For every test, there are multiple questions (ie 1 to many relationship)

In the tests table, I would like to have a column called "totalPoints" which is the sum of the points column for every row in Questions that is a child of Tests. Is there an array helper that can help? Suggestions on how to achieve? Thanks!

**Comment by melohagan** [comment, marked as answer]
Hi @dgagliardi 

Thank you for your post. I have added a tutorial to the Formulas doc page: https://docs.budibase.com/docs/formula#tutorial-test-scores-using-sum

Let me know if that helps!

**Reply by dgagliardi** [reply]
Wow @melohagan, that was a great help. Where does "pluck" come from? I can't find documentation on it anywhere. Is there a list of functions/expressions I can use in formulas? please share link

**Reply by melohagan** [reply]
Hi @dgagliardi 
All of the handlebars functions are Budibase specific. There are no docs for them, but if you scroll down the left-hand side of the bindings panel you'll be able to see all of the available helpers and a summary of use:

![Screenshot 2022-05-18 at 09 17 45](https://user-images.githubusercontent.com/101575380/168991882-647bda30-52b6-4fa9-8462-35d9ba38847a.png)
